### ISSUE TITLE
Support for new bulb model number

### ISSUE THREAD

**Issue by ChrisKingWebDev** [jdeath/Hubspace-Homeassistant#54]
Hi team. Great integration you have made here. It's working great with my slightly older bulbs, but the bulbs I just got this week have a different model number.

The new ones I have are 12A19060WRGBWH2. https://www.homedepot.ca/product/ecosmart-led-a19-e26-60-watt-equivalent-a-line-lamps-energy-saving-rgbw-2200k-6500k-2-pack-/1001773506

All the web results for this model number are for homedepot.ca, so it might be unique to Canada. They are the functionally and visually the same as my old ones which have model number 11A19060WRGBWH1. The only ones were also sold in a 2-pack, but maybe internally had the old model numbers.

I'm also happy to make a pull request for this, if it will help get it in faster.

**Comment by ChrisKingWebDev** [comment]
The output of the TestHubspace.py script is too long to post here, I have 12 devices hooked up.

**Comment by ChrisKingWebDev** [comment]
[TestHubspaceOutput.txt](https://github.com/jdeath/Hubspace-Homeassistant/files/12718261/TestHubspaceOutput.txt)
Here it is a text file.

**Comment by jdeath** [comment]
Thanks. Unfortunately, the nice people at hubspace make our lives difficult by leaving the model number field blank on new devices. I'll try to post something that works.

**Comment by jdeath** [comment]
Try latest light.py and hubspace.py. manually download and replace those files. Let me know if works.

**Comment by ChrisKingWebDev** [comment]
Thanks so much for writing this so quickly, but unfortunately it didn't work for me. I cloned the repo locally then copied the files through the Home Assistant terminal. I reinstalled the bulbs with a new name in Hubspace, but they still show up without a model name, and are limited to turning on and off. 

![Screen Shot 2023-09-27 at 4 25 52 PM](https://github.com/jdeath/Hubspace-Homeassistant/assets/6982185/fc20df4b-162b-4e42-ae87-7cdb73e6564a)

It really looks like the code should work, so I'm not sure what to suggest. I'm happy to run some debugging but I'm not sure how to get at it.

**Comment by jdeath** [comment]
Found a bug. Try latest Hubspace.py

**Comment by ChrisKingWebDev** [comment]
Shoot, it's still not working for me. I got the newest code, rebooted the system, and readded the bulbs with new names. Still showing up as it did before, with no model name.

**Comment by ChrisKingWebDev** [comment]
I was able to trace the problem and found that the getChildId wasn't firing on line 326 of lights.py. I changed this line 320 from `if None in (childId, model, deviceId, deviceClass):` to `if None in (childId, model, deviceId, deviceClass) or "" in (childId, model, deviceId, deviceClass):` which allowed it to get into getChildId. Now it's showing up with correct Model and I can change the color in the interface.
